cmd/lassie/version.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+
+	lcli "github.com/filecoin-project/lassie/cli"
+	"github.com/urfave/cli/v2"
+)
+
+var version string // supplied during build with `go build -ldflags="-X main.version=v0.0.0"`
+
+type Version struct {
+	Version string `json:"version"`
+}
+
+var versionCmd = &cli.Command{
+	Name:      "version",
+	Before:    before,
+	Usage:     "Prints the version and exits",
+	UsageText: "lassie version",
+	Flags: []cli.Flag{
+		lcli.FlagVerbose,
+	},
+	Action: versionCommand,
+}
+
+func versionCommand(cctx *cli.Context) error {
+	if version == "" {
+		log.Warn("executable built without a version")
+		log.Warn("set version with `go build -ldflags=\"-X main.version=v0.0.0\"")
```

Comment:
do we need to adapt our release build github workflow to do this?

Reply:
Yes, I was meaning to look into that and haven't yet. Ideally we have that build flag use the version.json to set that value.